src/__stories__/Customization.stories.mdx

```diff
@@ -1,12 +1,18 @@
-import { Meta, Story } from '@storybook/addon-docs/blocks'
-import { colors } from '..'
-import Box from '../..'
+import { Meta } from '@storybook/addon-docs/blocks'
+import { Box } from '..'
+import { colors } from '../theme'
 
-<Meta title="Theme/Colors" />
+<Meta title="Customization" />
 
-# Colors
+# Theming
 
-<Story name="Palette">
+Scaleway UI is based on `@emotion/react` library. It allows you to customize the theme as you need.
+
+## Palette
```

Comment:
```suggestion
## Colors
```

I think it might be better to put simple title so if later a user search for colors in documentation search bar he will probably type `colors` and not `palette`. I'm totally open for discussion about that subject